clients/unshaded/src/main/java/tachyon/client/netty/NettyRemoteBlockReader.java

```diff
@@ -38,11 +39,13 @@
 /**
  * Read data from remote data server using Netty.
  */
-public final class NettyRemoteBlockReader implements RemoteBlockReader {
+public final class NettyRemoteBlockReader implements Closeable, RemoteBlockReader {
```

Comment:
Right now, all that we do in `#close()` is release the `NettyDataBuffer`, so now we only need `#close()` for  `NettyRemoteBlockReader`. I think I added a TODO for investigating this.